docs/rtfd-requirements.txt

```diff
@@ -1 +1,3 @@
 pyquil
+sphinx-autodoc-typehints
+nbsphinx
```

Comment:
Both of these appear in `../requirements.txt`, although the first is spelled `sphinx_autodoc_typehints` (underscores) there. Do they need to be repeated here?